CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ include_directories(
     ${PROJECT_SOURCE_DIR}/hyrise/src/lib/                                                           #
     ${PROJECT_SOURCE_DIR}/hyrise/third_party/sql-parser/src                                         #
     ${TBB_INCLUDE_DIR}                                                                              #
-    ${Boost_INCLUDE_DIRS}                                                                           #
+    ${Boost_INCLUDE_DIR}                                                                           #
```

Comment:
Aaaah! There is `Boost_INCLUDE_DIR` and `Boost_INCLUDE_DIRS`, the first being a cache variable the latter a result variable. https://cmake.org/cmake/help/latest/module/FindBoost.html

Reply:
To be honest, I don't understand the difference. Will check on Linux as well before mergin.